src/Stripe.net/Infrastructure/Requestor.cs

```diff
@@ -120,7 +117,7 @@ private static async Task<StripeResponse> ExecuteRequestAsync(HttpRequestMessage
 
         internal static HttpRequestMessage GetRequestMessage(string url, HttpMethod method, RequestOptions requestOptions)
         {
-            requestOptions.ApiKey = requestOptions.ApiKey ?? StripeConfiguration.GetApiKey();
+            requestOptions.ApiKey = requestOptions.ApiKey ?? StripeConfiguration.ApiKey;
```

Comment:
Okay minor but need to flag: there's no thread safety and such, which I think is okay, but removing a getter prevents from ever adding this right? Though I _guess_ it still has a getter/setter associated with it?

Reply:
I'm not sure I follow. The only thing that changed with regards to the global API key is that `StripeConfiguration.GetApiKey()` is now `StripeConfiguration.ApiKey` and `StripeConfiguration.SetApiKey(newKey)` is now `Stripe.Configuration.ApiKey = newKey`.

Changing the global API was never thread-safe. Users that want to use different API keys in a multi-threaded environment must set the API key per request via `RequestOptions`.